### PR TITLE
Remove beta flags for OPA

### PR DESCRIPTION
**Pull request by mrinalirao**
### What
This commit removes the beta flag for OPA as it is now generally available.

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [x] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [x] Description links to related pull requests or issues, if any.

#### Content
- [x] Redirects have been added for moved, renamed, or deleted pages. This requires a separate PR in the [`terraform-website` repository](https://github.com/hashicorp/terraform-website) `redirects.next.js` file.
- [x] API documentation and the API Changelog have been updated. 
- [x] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [x] Pages with related content are updated and link to this content when appropriate.
- [x] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [x] New pages have metadata (page name and description) at the top.
- [x] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [x] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [x] UI elements (button names, page names, etc.) are bolded.
- [x] The Vercel website preview successfully deployed.

#### Reviews
- [x] I or someone else reviewed the content for technical accuracy.
- [x] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

File: website/docs/cloud-docs/api-docs/policies.mdx [hunk 1]
```diff
@@ -36,7 +36,7 @@ page_title: Policies - API Docs - Terraform Cloud
 
 # Policies API
 
--> **Note:** Sentinel and OPA policies are available in the [Terraform Cloud Team & Governance tier](https://www.hashicorp.com/products/terraform/pricing), and OPA policies are in beta. OPA Policies are not available in Terraform Enterprise.
+-> **Note:** Sentinel and OPA policies are available in the [Terraform Cloud Team & Governance tier](https://www.hashicorp.com/products/terraform/pricing).
```
Comment (cam-stitt) [inline website/docs/cloud-docs/api-docs/policies.mdx:39]:
```suggestion
-> **Note:** Sentinel and OPA policies are available in the [Terraform Cloud Team & Governance tier](https://www.hashicorp.com/products/terraform/pricing). OPA Policies are not available in Terraform Enterprise.
```

File: website/docs/cloud-docs/api-docs/policy-checks.mdx [hunk 2]
```diff
@@ -36,7 +36,7 @@ page_title: Policy Checks - API Docs - Terraform Cloud
 
 # Policy Checks API
 
--> **Note:** Sentinel and OPA policies are available in the [Terraform Cloud Team & Governance tier](https://www.hashicorp.com/products/terraform/pricing), and OPA policies are in beta. OPA policies are not available in Terraform Enterprise.
+-> **Note:** Sentinel and OPA policies are available in the [Terraform Cloud Team & Governance tier](https://www.hashicorp.com/products/terraform/pricing).
```
Comment (cam-stitt) [inline website/docs/cloud-docs/api-docs/policy-checks.mdx:39]:
```suggestion
-> **Note:** Sentinel and OPA policies are available in the [Terraform Cloud Team & Governance tier](https://www.hashicorp.com/products/terraform/pricing). OPA Policies are not available in Terraform Enterprise.
```

File: website/docs/cloud-docs/api-docs/policy-set-params.mdx [hunk 3]
```diff
@@ -36,7 +36,7 @@ page_title: Policy Set Parameters - API Docs - Terraform Cloud
 
 # Policy Set Parameters API
 
--> **Note:** Sentinel and OPA policies available in the [Terraform Cloud Team & Governance tier](https://www.hashicorp.com/products/terraform/pricing), and OPA policies are in beta. OPA policies are not available in Terraform Enterprise.
+-> **Note:** Sentinel and OPA policies available in the [Terraform Cloud Team & Governance tier](https://www.hashicorp.com/products/terraform/pricing).
```
Comment (cam-stitt) [inline website/docs/cloud-docs/api-docs/policy-set-params.mdx:39]:
```suggestion
-> **Note:** Sentinel and OPA policies available in the [Terraform Cloud Team & Governance tier](https://www.hashicorp.com/products/terraform/pricing). OPA Policies are not available in Terraform Enterprise.
```

File: website/docs/cloud-docs/api-docs/policy-sets.mdx [hunk 4]
```diff
@@ -36,7 +36,7 @@ page_title: Policy Sets - API Docs - Terraform Cloud
 
 # Policy Sets API
 
--> **Note:** Sentinel and OPA policies are available in the [Terraform Cloud Team & Governance tier](https://www.hashicorp.com/products/terraform/pricing), and OPA policies are in beta. OPA policies are not available in Terraform Enterprise.
+-> **Note:** Sentinel and OPA policies are available in the [Terraform Cloud Team & Governance tier](https://www.hashicorp.com/products/terraform/pricing).
```
Comment (cam-stitt) [inline website/docs/cloud-docs/api-docs/policy-sets.mdx:39]:
```suggestion
-> **Note:** Sentinel and OPA policies are available in the [Terraform Cloud Team & Governance tier](https://www.hashicorp.com/products/terraform/pricing). OPA Policies are not available in Terraform Enterprise.
```

**Review by cam-stitt — CHANGES_REQUESTED** [review]
Let's leave `OPA Policies are not available in Terraform Enterprise.` in all the messages!

**Review by glennsarti — APPROVED** [review]

**Review by cam-stitt — APPROVED** [review]

**Pull request merged**